.travis.yml

```diff
@@ -30,6 +32,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - pip install anybox.testing.openerp # needed for testing account_invoice_line_sort
```

Comment:
@lBati, some tests in the module `account_invoice_line_sort` make use of this package.
```
ERROR openerp_test openerp.modules.module: Can not `import account_invoice_line_sort`.

Traceback (most recent call last):

File "/home/travis/odoo-8.0/openerp/modules/module.py", line 377, in get_test_modules

mod = importlib.import_module('.tests', modpath)

File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module

__import__(name)

File "/home/travis/build/OCA/account-invoicing/account_invoice_line_sort/tests/__init__.py", line 2, in <module>

from . import test_account_invoice_line_sort

File "/home/travis/build/OCA/account-invoicing/account_invoice_line_sort/tests/test_account_invoice_line_sort.py", line 24, in <module>

from anybox.testing.openerp import SharedSetupTransactionCase

ImportError: No module named anybox.testing.openerp
```